Tests/LibWeb/Layout/expected/multi-code-point-graphemes.txt

```diff
@@ -2,19 +2,19 @@ Viewport <#document> at (0,0) content-size 800x600 children: not-inline
   BlockContainer <html> at (0,0) content-size 800x600 [BFC] children: not-inline
     BlockContainer <body> at (8,16) content-size 784x83 children: not-inline
       BlockContainer <p> at (8,16) content-size 784x17 children: inline
-        frag 0 from TextNode start: 0, length: 11, rect: [8,16 20.3125x17] baseline: 13.296875
+        frag 0 from TextNode start: 0, length: 11, rect: [8,16 11.6875x17] baseline: 13.296875
             "🧑‍🚒"
         TextNode <#text>
       BlockContainer <(anonymous)> at (8,49) content-size 784x0 children: inline
         TextNode <#text>
       BlockContainer <p> at (8,49) content-size 784x17 children: inline
-        frag 0 from TextNode start: 0, length: 13, rect: [8,49 20.3125x17] baseline: 13.296875
+        frag 0 from TextNode start: 0, length: 13, rect: [8,49 11.6875x17] baseline: 13.296875
             "🏴‍☠️"
         TextNode <#text>
       BlockContainer <(anonymous)> at (8,82) content-size 784x0 children: inline
         TextNode <#text>
       BlockContainer <p> at (8,82) content-size 784x17 children: inline
-        frag 0 from TextNode start: 0, length: 25, rect: [8,82 20.3125x17] baseline: 13.296875
+        frag 0 from TextNode start: 0, length: 25, rect: [8,82 23.375x17] baseline: 13.296875
             "🧑‍🧑‍🧒‍🧒"
         TextNode <#text>
```

Comment:
do we know why this layout test is affected? this change does not seem related to text width calculation.